tests/drift/test_univariate_drift_methods.py

```diff
@@ -118,6 +118,16 @@ def test_wasserstein_both_continuous_analysis_with_neg_mean_medium_drift():  # n
     assert wass_dist == 3.99
 
 
+def test_wasserstein_both_continuous_analysis_estimate_with_out_of_reference_drift():  # noqa: D103
+    np.random.seed(1)
+    reference = pd.Series(np.random.normal(0, 1, 15_000), name='A')
+    analysis = pd.Series(np.random.normal(0, 10, 1_000_000), name='A')
```

Comment:
Maybe we should make this a bit more extreme? Current failure case was 500_000 vs. 9_000_000_000

Reply:
This was more about ensuring the calculation when going out of bounds was correct than testing memory consumption and handling. Still if you think it's worth it we can change it.

Reply:
As discussed: if this is  enough to make it go out of bounds, then that's fine by me. Not intended to do profiling, just to trigger the failure.